src/containers/pages/FocusInvestigation/map/active/index.tsx

```diff
@@ -58,18 +61,26 @@ import plansReducer, {
   Plan,
   reducerName as plansReducerName,
 } from '../../../../../store/ducks/plans';
+import structuresReducer, {
+  getStructuresFCByJurisdictionId,
+  InitialStructure,
+  InitialStructureGeoJSON,
+  reducerName as structuresReducerName,
+  setStructures,
+} from '../../../../../store/ducks/structures';
 import tasksReducer, {
   fetchTasks,
   getFCByPlanAndGoalAndJurisdiction,
-  getStructuresFCByJurisdictionId,
   reducerName as tasksReducerName,
   Task,
   TaskGeoJSON,
 } from '../../../../../store/ducks/tasks';
+import { jurisdictions } from '../../../../../store/ducks/tests/fixtures';
```

Comment:
@kahama94 - What's this being used for? Seems like fixtures should stick to tests?